src/components/Options/IQServer/IQServerOptionsPage.tsx

```diff
@@ -163,107 +175,278 @@ export default function IQServerOptionsPage(props: IqServerOptionsPageInterface)
+                                            </div>
+                                        )}
+                                    </div>
+                                </center>
+                            </div>
+                        </div>
+                        <div className='nx-grid-col nx-grid-col-33'>
+                            <div>
+                                <center>
+                                    <img
+                                        src='/images/sonatype-firewall-icon.png'
+                                        width='50'
+                                        alt=''
+                                        className={
+                                            extensionSettings.supportsFirewall === false ? 'dim-image' : 'not-dim'
+                                        }
+                                    />
+                                    <div>
+                                        {extensionSettings.supportsFirewall === false && (
+                                            <span>{_browser.i18n.getMessage('DOES_IQ_SUPPORT_FEATURE')} </span>
+                                        )}
+                                        <NxTextLink
+                                            external
+                                            href='https://www.sonatype.com/products/sonatype-repository-firewall'>
+                                            {_browser.i18n.getMessage('SONATYPE_FIREWALL')}
+                                        </NxTextLink>
+                                        {extensionSettings.supportsFirewall === false && <span>?</span>}
+                                        {extensionSettings.supportsFirewall === true && (
+                                            <div>
+                                                <NxTag color='turquoise'>{_browser.i18n.getMessage('SUPPORTED')}</NxTag>
+                                            </div>
+                                        )}
+                                    </div>
+                                </center>
+                            </div>
+                        </div>
+                    </NxGrid.Row>
+                    <NxDivider></NxDivider>
+                    <NxGrid.Row>
+                        <section className='nx-grid-col nx-grid-col--'>
                             <p className='nx-p'>
-                                <strong>2)</strong> {_browser.i18n.getMessage('OPTIONS_PAGE_SONATYPE_POINT_2')}
+                                <strong>1)</strong> {_browser.i18n.getMessage('OPTIONS_PAGE_SONATYPE_POINT_1')}
                             </p>
+
                             <div className='nx-form-row'>
-                                <NxFormGroup label={_browser.i18n.getMessage('LABEL_USERNAME')} isRequired>
+                                <NxFormGroup label={`URL`} isRequired>
```

Comment:
Label is not translated @maurycupitt

Reply:
It is in the messages.json - is that what you mean?

Reply:
It is not currently in messages - but should be?